src/main/scala/com/actian/spark_vector/colbuffer/decimal/DecimalIntColumnBuffer.scala

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
http://docs.scala-lang.org/style/files.html

"Another case is when multiple classes logically form a single, cohesive group, sharing concepts to the point where maintenance is greatly served by containing them within a single file".

Doesn't it seem to you that we're exactly in this case?